### ISSUE TITLE
`force-app` folder appears in the `ExperienceBundle` type in package.xml

### ISSUE THREAD

**Issue by gavignon** [scolladon/sfdx-git-delta#834]
Issue verification check:

- [ ] is the current repository fully deployable at the commit SHA provided with the 'from' parameter of the command?

## What is the problem?

---

When using latest version (5.38.0) the generated package.xml file has incorrect metadata members corresponding to package directories (for example: force-app).

### What is the parameter and the value you used with it?

sf sgd source delta -i .deltaIgnore -f {PREVIOUS_TAG} -t HEAD -a 59.0 --output 'output'

### What is the expected result?

output/package/package.xml with correct metadata types and members.

### What is the actual result?

output/package/package.xml with incorrect metadata types and members (for example "force-app" in the metadata type "ExperienceBundle"

`<types>
        <members>CustomerService</members>
        <members>force-app</members>
        <name>ExperienceBundle</name>
    </types>`

## Steps to reproduce

sf sgd source delta -i .deltaIgnore -f {PREVIOUS_TAG} -t HEAD -a 59.0 --output 'output'

## Execution context

---

**Operating System:** MacOS Sonoma

**yarn version:** 1.22.17

**node version:** v20.12.2

**git version:** 2.42.0

**sfdx version:** 2.36.8

**sgd plugin version:** 2.38.0

**Comment by scolladon** [comment]
Hi @gavignon !

Thanks for raising this issue and thanks for contributing in making this project better!

Do you know how to reproduce it? What is the minimum change I can do in a commit to have containing folder being listed in the `ExperienceBundle` type?
I'd like to fix it fast 👍

**Comment by gavignon** [comment]
@scolladon I have the issue on several repositories without any specific change. 
Each time it's added to another metadatatypes, for example I had the issue on an Analytics repository with multiple package directories and they all went to "EmailServicesFunction" metadata type, while no metadata with this type is present in the repository.

**Comment by scolladon** [comment]
The `EmailServicesFunction` issue should be fixed with the current `latest` and `latest-rc` version (`5.38.3`)
I'll try to reproduce the `force-app` folder issue ASAP
Thanks for your help 👍

**Comment by richcallear** [comment]
Also experiencing this bug this afternoon - unsure if it's coincidence or not but my delta does consist of some Digital Experience changes; but then this spurious ExperienceBundle called force-app is getting added too.

**Comment by richcallear** [comment]
> Also experiencing this bug this afternoon - unsure if it's coincidence or not but my delta does consist of some Digital Experience changes; but then this spurious ExperienceBundle called force-app is getting added too.

Just following up - I rolled back release by release to 5.36.0 but still had the same issue - then I jumped back to 5.13.0 and the issue is not present - so it feels like this has been in the codebase for a while; but it's really odd that 2 of us came across it within an hour of each other this afternoon.....

**Comment by scolladon** [comment]
Yes @richcallear, thanks for the analysis 🙏 

You can find details [here](https://github.com/scolladon/sfdx-git-delta/issues/830#issuecomment-2057784981) of why it occurs

TL;DR;
If you are impacted by this bug you should use the `stable` channel (currently `v5.34.0`)
I'm currently reproducing and debugging
I'll ship the fix ASAP
